vms/platformvm/utxo/handler.go

```diff
@@ -166,10 +166,12 @@ func (h *handler) Spend(
 	// Minimum time this transaction will be issued at
 	now := uint64(h.clk.Time().Unix())
 
-	ins := []*avax.TransferableInput{}
-	returnedOuts := []*avax.TransferableOutput{}
-	stakedOuts := []*avax.TransferableOutput{}
-	signers := [][]*secp256k1.PrivateKey{}
+	var (
+		ins          = []*avax.TransferableInput(nil)
```

Comment:
q: I believe most of p-chain txs do consume inputs (I assume e.g. import/export are the most common, but I may be wrong). If this is true wouldn't be more efficient to allocate ins as we currently do? (Getting an estimate for capacity would be great too)